### PR TITLE
fixed typos in readme, fixed dependence on zcat which doesn't work we…

### PR DESCRIPTION
**Pull request by yfarjoun**
…ll on osx

### CODE REVIEW

File: ArraysAmbiguousSNPMiscallingMar2019/README.md [hunk 1]
```diff
@@ -8,16 +8,13 @@ Example Invocations:
     ./isVcfAffected.sh affected.vcf 
     The file affected.vcf (of array type: 'MEG_AllofUs_20002558X351448_A1') is affected by the Arrays Ambiguous SNP Bug
 
-    ./isVcfAffected.sh affected.vcf.gz 
-    The file affected.vcf.gz (of array type: 'MEG_AllofUs_20002558X351448_A1') is affected by the Arrays Ambiguous SNP Bug
+    ./isVcfAffected.sh unaffected.vcf.gz 
+    The file unaffected.vcf.gz is NOT affected by the Arrays Ambiguous SNP Bug
```
Comment (gbggrant) [inline ArraysAmbiguousSNPMiscallingMar2019/README.md:12]: I believe it DOES emit the string '(of array type: foo)' here.

File: ArraysAmbiguousSNPMiscallingMar2019/isVcfAffected.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ if [ ! -f $1 ]; then
 fi
 
 if [[ $1 == *.gz ]]; then
-	zcat $1 | sed '/^#CHROM/q;' | grep "extendedIlluminaManifestVersion=1.[012]" > /dev/null
+	gunzip -c $1 | sed '/^#CHROM/q;' | grep "extendedIlluminaManifestVersion=1.[012]" > /dev/null
```
Comment (gbggrant) [inline ArraysAmbiguousSNPMiscallingMar2019/isVcfAffected.sh:14]: good to know about -c

**Review by gbggrant — APPROVED** [review]

**Comment by gbggrant** [comment]
The readme conflicts with master.

File: ArraysAmbiguousSNPMiscallingMar2019/README.md (hunk 1 above)
Comment (yfarjoun) [reply ArraysAmbiguousSNPMiscallingMar2019/README.md:12]: i copied this from my console:

```
$ ./isVcfAffected.sh affected.vcf.gz 
The file affected.vcf.gz (of array type: 'MEG_AllofUs_20002558X351448_A1') is affected by the Arrays Ambiguous SNP Bug

$ ./isVcfAffected.sh unaffected.vcf.gz 
The file unaffected.vcf.gz is NOT affected by the Arrays Ambiguous SNP Bug
```

Comment (yfarjoun) [reply ArraysAmbiguousSNPMiscallingMar2019/README.md:12]: feel free to fix the script though, if you want it to be different... :-)

**Pull request merged**